WordPress/Classes/ViewRelated/Post/EditImageDetailsViewController.m

```diff
@@ -285,20 +305,35 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
     UITableViewCell *cell;
 
-    if (sec == ImageDetailsSectionThumb) {
-        // cofigure the thumb
-        cell = [self thumbCellForIndexPath:indexPath];
-    } else if (sec == ImageDetailsSectionDetails) {
-        // return a textfield cell
-        cell = [self detailCellForIndexPath:indexPath];
-    } else if (sec == ImageDetailsSectionDisplay) {
-        // display cells are textfields or normal cell
-        cell = [self displayCellForIndexPath:indexPath];
+    switch (sec) {
```

Comment:
Good that you refactor to a switch! 
I know that was already like that, but maybe rename `sec` to `section`?